pipelines/nlu-extension-build.yml

```diff
@@ -0,0 +1,74 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - /extensions/
+    - /pipelines/extension-build.yml
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - /extensions/
+    - /pipelines/extension-build.yml
+
+stages:
+- stage: Build_private
+  jobs:
```

Comment:
Since this is a single stage build and job, can we remove them and stick with the default stage / job?

Reply:
Removed stages